docs/DeveloperGuide.adoc

```diff
@@ -364,31 +364,36 @@ _Figure 3.1.2g: `UndoRedoStack` activity diagram_
+These two commands require one compulsory argument which is the amount that is borrowed/paid back. Indicating the index (as listed in the person list panel on the left side of the application window) of the person who borrowed or paid back money is optional. If no index is specified, the command will be executed for the person that is currently selected in the person list.
+The arguments (index and amount borrowed/paid back) are separated by a whitespace instead of special prefixes (e.g. prefix `p/` used for phone number). Hence, the `String#split` method is used to tokenize the input using a single whitespace as the delimiter. As seen in Figure 3.2.1, the tokenized inputs (index and amount borrowed) are then converted to their appropriate `Object` types and supplied as arguments to the `BorrowCommand` constructor. Input for the `payback` command is tokenized in the same manner.
 
-When `BorrowCommand` is executed in `LogicManager`, the `Debt` attribute in the target `Person` object is updated in the `AddressBook` class in the `Model` component. A new `Person` object is created with the debt amount updated to reflect the new total debt. The target `Person` object is then replaced with this new `Person` object.
+The `BorrowCommand` and `PaybackCommand` are executed in `LogicManager`.
+`BorrowCommand` updates the `debt` and `totalDebt` attributes in the target `Person` object in the `AddressBook` class (which is in the `Model` component). A new `Person` object is created with the updated `debt` and `totalDebt` amount. The target `Person` object is then replaced with this new `Person` object.
+`PaybackCommand` updates the target person's debt in the same manner as the `BorrowCommand`. However, only the `debt` attribute is updated.
 
 [NOTE]
 `DateBorrow` field in the new `Person` object needs to be updated to match the `DateBorrow` field in the target `Person` object. This is because the date when the `Person` object was created is set as the date for the `DateBorrow` field.
 
-The following sequence diagram, Figure 3.2.1, shows further details of the interaction between the `Logic` and `Model` component when the borrow command is executed:
+The following sequence diagram, Figure 3.2.1, shows further details of the interaction between the `Logic` and `Model` component when the `borrow` command is executed:
 
 image::BorrowCommandSequenceDiagram.png[width="800"]
 _Figure 3.2.1 : Sequence diagram of how the `Borrow` command works_
 
+The `payback` command works in the same way as the `borrow` command.
```

Comment:
Can consider changing to "in a similar way" since there is a small difference between the two (one updates total debt and the other doesn't)